src/client/theme-default/components/VPDocAsideOutline.vue

```diff
@@ -45,16 +40,15 @@ function handleClick({ target: el }: Event) {
 
         <ul class="root">
           <li
-            v-for="{ text, link, children, hidden } in resolvedHeaders"
-            v-show="!hidden"
+            v-for="{ title, link, children } in page.headers"
```

Comment:
Just a heads up, @vuejs/theme would also need to update this (`text` -> `title`).

Reply:
Yeah, but not necessary. Here we changed it to `title` because we don't need an extra `resolvedHeaders` transform. The type of `Header` is still compatible.